documentation/modules/oauth/con-mapping-keycloak-authz-services-to-kafka-model.adoc

```diff
@@ -0,0 +1,92 @@
+[id="con-mapping-keycloak-authz-services-to-kafka-model_{context}"]
+= Mapping Keycloak Authorization Services to the Kafka authorization model
+
+[role="_abstract"]
+
+To grant Kafka permissions to user accounts or service accounts, use Keycloak Authorization Services rules on the OAuth client that represents the Kafka Broker.
```

Comment:
I think we should introduce this section by saying how we do the _mapping_. That is, _we use the Kafka authorization model as a basis for defining the Keycloak roles and resources that will control access to Kafka..._

Reply:
That's great, thank you.

"The Kafka authorization model is used as a basis for defining the Keycloak roles and resources that will control access to Kafka."